buildSrc/build.gradle

```diff
@@ -116,6 +116,7 @@ dependencies {
   api 'org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10'
```

Comment:
Sorry for being peeky, it seems like we have a number of `'org.jetbrains.kotlin` deps (like here and [in the](https://github.com/opensearch-project/OpenSearch/pull/4963/files#diff-6ffeac346fb766ef182207bf47e9f4472bae38bc46196eb14bb690de71c4e4cdR54) , could we put the version in `versions.properties` instead?

Reply:
Makes sense! Done.